RELEASE.md

```diff
@@ -64,177 +72,304 @@ A release for the bpfman project is comprised of the following major components:
+- Release note details.
 
-Often times cutting a release candidate is a great way to test any changes to
-our release infrastructure before cutting an official release. Make sure release
-candidate versions contain an `rc` suffix i.e `0.4.0-rc1`.  This is a lighter
-weight process meaning many of the versioned manifests do not necessarily need
-to be created.
-
-- Open an update PR that:
-    - Adds a new changelog for the release
-    - Updates the [Cargo.toml](https://github.com/bpfman/bpfman/blob/main/Cargo.toml) version for the workspace:
-        - `version = "x.x.x"`
-        - `bpfman = { version = "x.x.x", path = "./bpfman" }"`
-        - `bpfman-api = { version = "x.x.x", path = "./bpfman-api" }`
+To simplify the generation of the release notes details, we recommend using the
+GitHub release page as described below.  Note that we only use the release page
+to generate a starting point for the release notes, and don't actually create a
+tag or do a release from it. 
+
+1. Go to https://github.com/bpfman/bpfman/releases.
+2. Push the "Draft a new release" button.
+3. Enter the new release number in the "Choose a tag" pull-down.
+4. Choose the most recent release in the "Previous tag" pull-down.
+5. Push the "Generate release notes" button.
+
+The automatically generated output will likely need to be reorganized and
+cleaned up a bit, but it provides a good starting point.
+
+### Standard Release from Main Branch
+
+This section describes the standard release process used when making a release
+from the main branch and may be used for major, minor, or patch releases.  As
+mentioned above, we first complete the release for `bpfman` and then follow that
+up with a release for `bpfman-operator`.
+
+#### bpfman Release
+
+- Create a new branch in your `bpfman` fork named, for example
+  `<githubuser>/release-x.y.z`, and use the new branch in the upcoming steps.
+- Make the following changes
+    - Add a new changelog for the release using the process described in
+      [Generating Release Notes](#generating-release-notes).
+    - Update the
+      [Cargo.toml](https://github.com/bpfman/bpfman/blob/main/Cargo.toml)
+      version for the workspace:
+        - `version = "x.y.z"`
+        - `bpfman = { version = "x.y.z", path = "./bpfman" }"`
+        - `bpfman-api = { version = "x.y.z", path = "./bpfman-api" }`
         - Note: `bpfman-csi` does not need to be updated.
-    - Runs `cargo generate-lockfile`
-    - Updates the bpfman-operator version in it's [Makefile](https://github.com/bpfman/bpfman-operator/blob/main/Makefile):
-        - `VERSION ?= x.x.x`
-- Make sure CI is green and merge the update PR.
-- Create a tag using the `HEAD` of the `main` branch. This can be done using the `git` CLI or
-  Github's [release][release] page.
-- Tag the release using the commit on `main` where the changelog update merged.
-  This can  be done using the `git` CLI or Github's [release][release]
+    - Run `cargo generate-lockfile`
```

Comment:
- Update the bpfman version in the [examples/Makefile][examples-makefile]:
    - `VERSION ?= x.y.z`

And add link at bottom of file.

Reply:
Done.